workflows/sars-cov-2-variant-calling/sars-cov-2-consensus-from-variation/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.2.3] 2022-01-20
+
+### Automatic update
+- `toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_genomecoveragebed/2.29.2` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/bedtools/bedtools_genomecoveragebed/2.30.0`
+- `toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_filter/4.3+t.galaxy1` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_filter/4.3.0`
+- `toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_extractFields/4.3+t.galaxy0` was updated to `toolshed.g2.bx.psu.edu/repos/iuc/snpsift/snpSift_extractFields/4.3.0`
```

Comment:
The version strings of the snpsift (and snpeff) tools are historically problematic, and the bot falls into the trap and recommends updating to an *older* version.

Reply:
Yes, I resolved not to fix that. We can also put the tools on the skiplist if desired.

Thanks for taking a look at the PRs!